diff --git a/node_modules/@uniswap/sdk-core/dist/sdk-core.cjs.development.js b/node_modules/@uniswap/sdk-core/dist/sdk-core.cjs.development.js
index 29f7a7c..6d6e691 100644
--- a/node_modules/@uniswap/sdk-core/dist/sdk-core.cjs.development.js
+++ b/node_modules/@uniswap/sdk-core/dist/sdk-core.cjs.development.js
@@ -584,7 +584,7 @@ var Ether = /*#__PURE__*/function (_NativeCurrency) {
   _inheritsLoose(Ether, _NativeCurrency);
 
   function Ether(chainId) {
-    return _NativeCurrency.call(this, chainId, 18, 'ETH', 'Ether') || this;
+    return _NativeCurrency.call(this, chainId, 18, 'MON', 'Monad') || this;
   }
 
   Ether.onChain = function onChain(chainId) {
diff --git a/node_modules/@uniswap/sdk-core/dist/sdk-core.esm.js b/node_modules/@uniswap/sdk-core/dist/sdk-core.esm.js
index 01b2fea..d739a1c 100644
--- a/node_modules/@uniswap/sdk-core/dist/sdk-core.esm.js
+++ b/node_modules/@uniswap/sdk-core/dist/sdk-core.esm.js
@@ -582,7 +582,7 @@ var Ether = /*#__PURE__*/function (_NativeCurrency) {
   _inheritsLoose(Ether, _NativeCurrency);
 
   function Ether(chainId) {
-    return _NativeCurrency.call(this, chainId, 18, 'ETH', 'Ether') || this;
+    return _NativeCurrency.call(this, chainId, 18, 'MON', 'Monad') || this;
   }
 
   Ether.onChain = function onChain(chainId) {
